e2e/monitor/README.md

```diff
@@ -0,0 +1,45 @@
+# Localnet Monitor
+
+The Localnet Monitor is a small program that simply polls localnet for values
+that we want to test against.  
+
+## Prerequisites
+
+* Go 1.22+
+* `docker` available in your cli
+
+## Running
+
+You can run the Localnet Monitor like so, this will read from localnet
+and print a table that refreshes every 1 second.
+
+Make sure you have localnet running:
+
+from the root of the repo:
+```
+docker compose -f ./e2e/docker-compose.yml down -v --remove-orphans
```

Comment:
Is the `--remove-orphans` option necessary?  I've very rarely needed to use that with compose?

Reply:
hey @jcvernaleo , it's not _necessary_, but it is good practice.

it's defined as:
> Remove containers for services not defined in the Compose file

for example, if we had a postgres container running that wasn't defined in `docker-compose.yml`, and we didn't include the `--remove-orphans` flag, it could still be running when we try to run this orchestrations.

the `--remove-orphans` flag is another step that helps us ensure that we have a clean environment before running the services.